lib/clearance/password_strategies/blowfish.rb

```diff
@@ -3,6 +3,7 @@
 
 module Clearance
   module PasswordStrategies
+    # @deprecated Use BCrypt or `clearance-deprecated_password_strategies` gem
```

Comment:
Maybe `Use {Clearance::PasswordStrategies::BCrypt}`, i.e. link to the actual module?